docs/development/randomness-and-reproducibility.rst

```diff
@@ -0,0 +1,99 @@
+Randomness and Reproducibility
+==============================
+
+``respy`` embraces randomness to study individual behavior under risk. At the same time,
+it is crucial to make results reproducible. To build a reproducible model, users must
+define three seeds for the solution, simulation and estimation of the model in the
+options.
+
+.. code-block:: python
+
+    options = {"solution_seed": 1, "simulation_seed": 2, "estimation_seed": 3}
```

Comment:
Maybe 1, 2, 3 is not a good example then. Because then we basically use the same seeds with one lag. How often do we increment? Maybe it would be good to assert somewhere that the seeds are far enough apart that we never reach the same seed again. 

Or even simpler: Why not just stick to one seed and then use that seed for simulation, that seed + k for estimation and that seed + 2k for solution (choosing k large enough to not get any duplication). That it also becomes harder to cherry pick seeds.

Reply:
I think @janosg 's suggestions of automatic increments is a good default. However, please make sure a more fine-grained control is still possible by setting seeds for each broad component (solution, simulation, estimation). I found it very useful to learn about the impact of randomness on each component separately.